include/lsst/meas/base/Blendedness.h

```diff
@@ -42,13 +42,14 @@ class BlendednessControl {
             doOld, bool,
             "Whether to compute HeavyFootprint dot products (the old deblend.blendedness parameter)");
 
-    LSST_CONTROL_FIELD(doFlux, bool, "Whether to compute quantities related to the Gaussian-weighted flux");
+    LSST_CONTROL_FIELD(doFlux, bool,
+                       "Whether to compute quantities related to the Gaussian-weighted instFlux");
```

Comment:
Is there any possibility that we'll want a flag that controls computation of non-inst Gaussian flux and/or non-Gaussian flux of either kind? If so, this name is worrisome. I hope there's no danger, as I suspect it's a nuisance to change.